README.md

```diff
@@ -259,6 +259,15 @@ Default: `['passwd', 'password', 'secret', 'confirm_password', 'password_confirm
 Default: `[]`
 </dd>
 
+<dt>showUnhandledErrors
+</dt>
+<dd>Whether or not to show unhandled errors on the console before sending them to Rollbar.  By default, unhandled errors will be consumed and silently sent to Rollbar. 
+
+Default: `false`
```

Comment:
Couple of things:

- Is this already happening for uncaught errors? https://github.com/rollbar/node_rollbar/blob/master/rollbar.js#L264
- If an error is passed into `handleError()` it is done so either by directly calling `handleError()` by the user or via one of the uncaught error handlers. For both of those cases it's really up to the user to tell us to print something out. So if the caller is already doing this we wouldn't want to default to printing them out a second time. Hence, defaulting to `false`. @brianr